bokeh/plotting/helpers.py

```diff
@@ -426,7 +453,9 @@ def _glyph_function(glyphclass, extra_docs=None):
 
     def func(self, *args, **kwargs):
 
+        # Process legend kwargs and remove legend before we get going
```

Comment:
This comment doesn't introduce much information over function name. Perhaps this should be a part of the given function's docstring, if at all needed.